src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.svelte

```diff
@@ -8,7 +8,6 @@
 
   export let data;
   $: ({ fireWeatherData, pageData, error = null } = data);
-  $: description = pageData?.description;
```

Comment:
I think since `pageData` is already reactive, we don't need `description` (or `title`) to also be reactive. Unless I'm missing something here?

Reply:
You're probably right. My memory of some of the reactivity stuff in Svelte was foggy, so I just defaulted to the greedy case, but I think that's unnecessary

Reply:
nb4 [SvelteKit 5's runes](https://vercel.com/blog/whats-new-in-svelte-5#runes) drastically change how all of this works 😵